custom-namespaces.yaml

```diff
@@ -1108,3 +1108,36 @@ hubs:
       type: table_view
       tables:
         - table: mozdata.hubs.active_subscription_ids
+subscription_platform:
+  pretty_name: Subscription Platform
+  owners:
+    - srose@mozilla.com
+  glean_app: false
+  views:
+    # Using `_v1` views to avoid breakages when v2 tables are deployed.
+    apple_subscriptions_v1:
+      type: table_view
+      tables:
+        - table: moz-fx-data-shared-prod.subscription_platform_derived.apple_subscriptions_v1
+    google_subscriptions_v1:
+      type: table_view
+      tables:
+        - table: moz-fx-data-shared-prod.subscription_platform_derived.google_subscriptions_v1
+    stripe_subscriptions_v1:
+      type: table_view
+      tables:
+        - table: moz-fx-data-shared-prod.subscription_platform_derived.stripe_subscriptions_v1
+  explores:
+    # Using `_v1` explores to avoid breakages when v2 tables are deployed.
+    apple_subscriptions_v1:
+      type: table_explore
```

Comment:
Worth noting that this is the first use of the new `table_explore` explore type added in #682.